src/pages/groupComparison/GroupComparisonMutationsTabPlot.tsx

```diff
@@ -0,0 +1,148 @@
+import * as React from 'react';
+import { observer } from 'mobx-react';
+import { action, computed, observable, makeObservable } from 'mobx';
+import autobind from 'autobind-decorator';
+import Loader from 'shared/components/loadingIndicator/LoadingIndicator';
+import { convertToMutationMapperProps } from 'shared/components/mutationMapper/MutationMapperServerConfig';
+import { getGenomeNexusHgvsgUrl } from 'shared/api/urls';
+import { getServerConfig } from 'config/config';
+import GroupComparisonMutationMapper from './GroupComparisonMutationMapper';
+import { Mutation } from 'cbioportal-ts-api-client';
+import MutationMapperToolStore from 'pages/staticPages/tools/mutationMapper/MutationMapperToolStore';
+import GroupComparisonStore from './GroupComparisonStore';
+import _ from 'lodash';
+import {
+    numberOfLeadingDecimalZeros,
+    formatPercentValue,
+} from 'cbioportal-utils';
+import { AxisScale } from './AxisScaleSwitch';
+import { MakeMobxView } from 'shared/components/MobxView';
+import { remoteData } from 'cbioportal-frontend-commons';
+import { countUniqueMutations } from 'shared/lib/MutationUtils';
+
+interface IGroupComparisonMutationsTabPlotProps {
+    store: GroupComparisonStore;
+    mutations?: Mutation[];
+    filters?: any;
+}
+
+@observer
+export default class GroupComparisonMutationsTabPlot extends React.Component<
+    IGroupComparisonMutationsTabPlotProps,
+    {}
+> {
+    @observable public showPercent = true;
+    constructor(props: IGroupComparisonMutationsTabPlotProps) {
+        super(props);
+        makeObservable(this);
+    }
+
+    private mutationMapperToolStore = remoteData<MutationMapperToolStore>({
+        await: () => [this.props.store.mutationsByGroup],
```

Comment:
we don't need to await here because we should be able to assume that the property is loaded